includes/discovery/fdb-table/vrp.inc.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * vrp.inc.php
+ *
+ * Discover FDB data with HUAWEI-L2MAM-MIB
```

Comment:
Could you update your copyright to use this template please:

https://docs.librenms.org/#Developing/Licensing/#copyright

Reply:
Done